core/new-gui/src/app/dashboard/user/component/user-workflow/user-workflow-list-item/dashboard-workflow-entry-view-model.ts

```diff
@@ -0,0 +1,5 @@
+import { DashboardWorkflowEntry } from "../../../type/dashboard-workflow-entry";
+
+export default interface DashboardWorkflowEntryViewModel extends DashboardWorkflowEntry {
+  checked: boolean;
+};;;;;;;;;;
```

Comment:
I'm not sure why `yarn format:fix` added `;;;;;;;;` here. It is failing CI check without this

Reply:
I don't think this is the correct behavior... Can you find out the root cause?

Reply:
Fixed it by editing the eslint file, according to [this](https://stackoverflow.com/questions/55642963/eslint-or-prettier-is-adding-2-semi-colons-to-the-end-of-a-line-in-vscode)